sklearn/preprocessing/tests/test_data.py

```diff
@@ -1202,6 +1207,7 @@ def test_normalizer_l2():
         X = init(X_dense)
         X_norm = normalizer = Normalizer(norm='l2', copy=False).transform(X)
 
+        assert_equal(X_norm.dtype, X.dtype)
```

Comment:
Oh, I meant that this test will pass in master as well. Because the dype of `X` is `np.float64`. We want to check that the dtype of X is preserved. Hence I would add a separate test, testing that `normalize` preserves dtype.